backend/cmd/headlamp.go

```diff
@@ -307,6 +314,35 @@ func createHeadlampHandler(config *HeadlampConfig) http.Handler {
 		proxy.ServeHTTP(w, r)
 	})
 
+	//nolint:gosec,scopelint
```

Comment:
Reduced the `nolint` as much as possible. Most of them are because of the way existing code is structured. Fixing this will require a lot of yak shaving, so skipping this as of now as refactoring the backend is on my list of things to improve.

Reply:
Thanks. I've added an issue here https://github.com/kinvolk/headlamp/issues/855 for these specific cleanups.